lib/proxy/request.rb

```diff
@@ -20,6 +20,12 @@ def create_get(path, query={}, headers={})
       req
     end
 
+    def create_delete(path, query={}, headers={})
+      req = Net::HTTP::Delete.new(uri(path).path + "/" + query[:id])
```

Comment:
`tap` might come in handy here.

Reply:
It's the same as in all the other functions there...